test/runtests.jl

```diff
@@ -21,6 +21,7 @@ end
         @time @safetestset "Higher Order Reactions" begin include("reactionsystem_core/higher_order_reactions.jl") end
         @time @safetestset "Symbolic Stoichiometry" begin include("reactionsystem_core/symbolic_stoichiometry.jl") end
         @time @safetestset "Parameter Type Designation" begin include("reactionsystem_core/parameter_type_designation.jl") end
+        @time @safetestset "Functional Parameters" begin include("reactionsystem_core/functional_parameters.jl") end
```

Comment:
I would put this a lot lower down in the list. It seems less core than things like events or coupling equations.

Reply:
(of the `ReactionSyste`m structure block)